src/components/footer/index.js

```diff
@@ -50,30 +50,26 @@ class Footer extends Component {
                 <li className="mainli"> 
                   <div className="title-sm">PRODUCT</div>
                   <ul className="sub-menu">
-                    <li className="menu-item"><a href="http://example.com">Why use POKT</a></li>
-                    <li className="menu-item"><a href="http://example.com">Road Map?</a></li>
-                    <li className="menu-item"><a href="http://example.com">Tutorials</a></li>
+                    <li className="menu-item"><a href="https://docs.pokt.network/docs/what-is-pocket-network">Why use POKT</a></li>
+                    <li className="menu-item"><a href="https://www.pokt.network/">Road Map</a></li>
                   </ul>
                 </li>
                 <li className="mainli">
                   <div className="title-sm">RESOURCES</div>
                   <ul className="sub-menu">
-                    <li className="menu-item"><a href="http://example.com">WhitePaper</a></li>
-                    <li className="menu-item"><a href="http://example.com">Economic One Pager</a></li>
-                    <li className="menu-item"><a href="http://example.com">iOS SDK</a></li>
-                    <li className="menu-item"><a href="http://example.com">Android SDK</a></li>
-                    <li className="menu-item"><a href="http://example.com">JS SDK</a></li>
-                    <li className="menu-item"><a href="http://example.com">Press kit</a></li>
+                    <li className="menu-item"><a href="https://pokt.network/wp-content/uploads/2020/03/POCKET-NETWORK-WHITEPAPER-V0.3.0.pdf">WhitePaper</a></li>
```

Comment:
These should be on S3 as well so they don't decay

Reply:
https://pocket-network-assets.s3-us-west-2.amazonaws.com/pdfs/Pocket-Network-Whitepaper-v0.3.0.pdf

Economic one pager needs a different link: 
https://pocket-network-assets.s3-us-west-2.amazonaws.com/pdfs/Pocket-Network-Economic-Paper-v1.0.2.pdf